lib/driver.js

```diff
@@ -129,7 +130,10 @@ class XCUITestDriver extends BaseDriver {
       await this.wda.launch(sessionId);
 
       this.proxyReqRes = this.wda.proxyReqRes.bind(this.wda);
-      this.jwpProxyActive = true;
+      // if we keep proxy active then all call will route to proxy
```

Comment:
I didn't like this conditional thing but if we don't do it then all calls will go to XCUITestDriver instead of safari. We need to change setContext later to support Native functionality for safari browser (lower pro)